### ISSUE TITLE
Recompile 

### ISSUE THREAD

**Issue by ajayavgarde** [luferau/labview-offline-maps#2]
Hey, nice work on this. I want some more features in it where I can plot multiple vehicle paths on the map with different colors and I don't want to see the temperature value. So can you help me with how it can be done?

**Comment by luferau** [comment]
@ajayavgarde what you're asking for cannot be done with the current code. This is just an example. You must extend the functionality of the OfflineMaps.Controls.dll .NET control and further pass the necessary data from the LabVIEW. 
Do you have experience with C#/WinForms and LabVIEW?

**Comment by luferau** [comment]
@ajayavgarde p.s. also feel free to give this repository a star if you like it

**Comment by ajayavgarde** [comment]
no, I haven't worked on C#, but I have worked on LabVIEW.So it will be bit difficult for me work on it
